src/obfl-specification.html

```diff
@@ -1225,6 +1225,26 @@ <h4 id="L1023" class="include">block</h4>
             left at the top of a page.</dd>
           <dd>An integer in the range [1, 9].</dd>
           <dd>Default is 1.</dd>
+        <dt>display-when [optional]</dt>
+          <dd>An condition that determines whether the contents of the block are
+            rendered.</dd>
+          <dd>Default is true.</dd>
+          <dd class="markup note">All markers in this block and its descendant
+            elements will be processed, regardless of the outcome of the
+            condition. So it is possible to use a marker's value even when the
+            block is not rendered.</dd>
```

Comment:
Hmm, I got another idea. Why don't we disallow `marker` when `display-when` is present? It's easy to let markers be processed unconditionally, by simply putting them before or after the block.

```xml
<block>
   <marker class="pagenum" value="7"/>
   <block display-when="(! $starts-at-top-of-page)">
      Page number: 7
   </block>
</block>
```

Of course this only works if the marker does not happen somewhere in the middle.

The benefit of this more conservative approach is that if later a use case emerges for making markers conditional as well, it's compatible.

Reply:
I have no problems with this more conservative approach.
We have to make a choice and I don't really have a strong preference. 
Like I mentioned already, the idea of disregarding identifiers occurs already in OBFL.
Both approaches will work.

Reply:
After some discussion we agreed to allow markers and anchors even when a `display-when` is present.